interfaces/CMakeLists.txt

```diff
@@ -7,21 +7,21 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-if (NOT DISABLE_C_SHARED_INTERFACE)
```

Comment:
Where is DISABLE_C_SHARED_INTERFACE defined? In the main CMakeLists.txt file I was only finding HELICS_DISABLE_C_SHARED_LIB.

Reply:
this should be fixed, I will fix it in the next update